openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamHierarchy.tsx

```diff
@@ -189,7 +153,10 @@ const TeamHierarchy: FC<TeamHierarchyProps> = ({
           components={TABLE_CONSTANTS}
           data-testid="team-hierarchy-table"
           dataSource={data}
-          expandable={expandableConfig}
+          expandable={{
+            ...getTableExpandableConfig<Team>(),
+            ...expandableConfig,
+          }}
```

Comment:
```suggestion
          expandable={expandableConfig}
```

And just spread `getTableExpandableConfig` to `expamdableConfig` only